app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/ControllerOptionsView.java

```diff
@@ -161,10 +165,35 @@ private void setPointerMode(@WidgetManagerDelegate.PointerMode int value, boolea
         }
     }
 
+    private void showHandTrackingWarningIfNeeded(boolean enabled) {
+        if (enabled) {
+            mBinding.handtrackingSwitchWarning.setVisibility(GONE);
+        } else {
+            mBinding.handtrackingSwitchWarning.setVisibility(VISIBLE);
+
+            int highlightColor = getContext().getResources().getColor(R.color.spark);
```

Comment:
The background animation effect was a bit surprising the first time, because white text on light orange is not easy to read.

Maybe we could use a darker color, like `eggplant` or `void_color`?

Also, the color animation looks nicer if it ends on a transparent version of the initial color.

Something like this:

```java
int highlightColor = getContext().getResources().getColor(R.color.eggplant);
int targetColor = highlightColor & 0x00FFFFFF;  // set alpha bits to zero
```

Reply:
WRT the color that was made on purpose. I really did want users to really ready it, but perhaps it was too much :)